Umee-Bots/Stale-Price/src/utils.ts

```diff
@@ -0,0 +1,88 @@
+import { Interface } from "@ethersproject/abi";
+
+import { ethers, Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export interface AgentConfig {
+  threshold: number;
+  lendingPoolAddress: string;
+  umeeOracleAddress: string;
+}
+
+const UMEE_FUNCTIONS_ABI: string[] = [
+  "function getReservesList() external view returns (address[] memory)",
+  "function getSourceOfAsset(address asset) external view returns (address)",
+  "function latestTimestamp() external view returns (uint256)",
+];
+
+const EVENT_ABI = ["event AssetSourceUpdated(address indexed asset, address indexed source)"];
+
+const FUNCTIONS_INTERFACE = new Interface([...UMEE_FUNCTIONS_ABI, ...EVENT_ABI]);
+
+export interface AssetDataI {
```

Comment:
as AssetData Interface

Reply:
But you know that's an interface without the `I` in the name, right? If there was a class `AssetData` that implemented from an interface, the interface could be `IAssetData` or `AssetDataI`, otherwise I think just `AssetData` is better, wdyt?

Reply:
You are right, also I have `AgentConfig` without the `I` so It is better to make it consistent.